src/features/call/components/ParticipantMedia.module.css

```
@@ -28,21 +29,29 @@
 
-.surface[data-media-state='video'] .placeholder {
-  display: none;
-}
+.videoStatus {
+  display: flex;
+  justify-content: center;
+  align-items: center;
 
-.surface:not([data-media-state='audio']) .audioOnly {
-  display: none;
+  z-index: 9;
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)